widgetastic_manageiq/vm_reconfigure.py

```diff
@@ -0,0 +1,47 @@
+from widgetastic.widget import Table as VanillaTable, Text
+from widgetastic.xpath import quote
+from widgetastic_patternfly import BootstrapSelect, BootstrapSwitch, Button, Input
+
+
+class DisksButton(Button):
+
+    # Relative to DisksTable
+    BTN_CONTAINER_LOC = "../h3//td[contains(@align, 'right') and contains(@id, 'buttons_on')]"
+
+    def __locator__(self):
```

Comment:
This should be marked with a clear comment that under normal circumstances such constructs are criminal :)